win32/src/winapi/user32/resource.rs

```diff
@@ -75,6 +75,8 @@ fn load_bitmap(
         ResourceKey::Id(pe::RT::BITMAP as u32),
         name,
     )?;
+    let image = machine.mem().slice(hInstance..);
+    let buf = &image[buf];
```

Comment:
... and if find_resource returns Range<u32>, this can be something like `mem().slice(range)` or so?